src/iterative_telemetry/__init__.py

```diff
@@ -190,10 +190,18 @@ def send(
         impl(payload)
 
     def _send_daemon(self, payload):
-        cmd = (
-            f"import requests;requests.post('{self.url}',"
-            f"params={{'token':'{self.token}'}},json={payload})"
-        )
+        cmd = f"""
+import requests, logging
+try:
+    requests.post(
+        '{self.url}',
+        params={{'token':'{self.token}'}},
+        json={payload},
+        timeout=2
```

Comment:
I think it should be safe to make it 10+ seconds?

Reply:
I used the default from the timeout we already had for thread based or normal method of sending the request. I am not sure if we want to have timeout upto 10 seconds delay to close a program when program already failed for the user.

Reply:
> if we want to have timeout upto 10 seconds delay to close a program when program already failed for the user.

are we waiting for the process to exit / complete on the user program exit?

Reply:
Looks like we have `close_fds=True,` so, we won't have the blocking exit. Will update the timeout to 10 seconds.

Reply:
Well, `close_fds=True` does not close `stdin`, `stdout` and `stderr` streams. So it might end up blocking. But that's a separate issue.